backend/app/GraphQL/Mutations/AcceptAnswer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use Exception;
+use App\Models\Answer;
+use App\Models\Question;
+
+final class AcceptAnswer
+{
+    /**
+     * @param  null  $_
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args)
+    {
+        // TODO implement the resolver
+        try {
+            $question = Question::findOrFail($args['question_id']);
+            $answer = Answer::findOrFail($args['answer_id']);
+            $checkAccepted = $question->answers()->where('answers.is_correct', true)->exists();
+            if($checkAccepted){
+                return "Only one answer can be accepted";
+            }
+            $answer->is_correct = true;
+            $answer->save();
+            return "Answer was accepted successfully";
```

Comment:
```suggestion
            $question = Question::findOrFail($args['question_id']);
            
            $answer = Answer::findOrFail($args['answer_id']);
            $checkAccepted = $question->answers()->where('answers.is_correct', true)->exists();
            
            if($checkAccepted){
                return "Only one answer can be accepted";
            }
            
            $answer->is_correct = true;
            $answer->save();
            
            return "Answer was accepted successfully";
```
You can put spaces(Entr key) between lines of code if they have different purposes so that it will be easier to read